index.html

```diff
@@ -8,26 +8,58 @@
-    <div class="flex flex-1 py-3 md:mt-5 md:mx-20 items-center justify-between">
-        <div class="flex-initial">
-            <p class="font-medium text-lg">DevTeam</p>
-            <p class="text-gray-400 text-sm">Sustainable Education Foundation</p>
-        </div>
-        <div class="flex text-xs font-light space-x-5">
-            <a class="inline-block" href="https://github.com/sef-global" target="_blank">Github</a>
-            <a class="inline-block" 
-               href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-9w6v9rdu-uJGLrMKLTNFzQEVPC4sbOw" 
-               target="_blank">
-                 Slack
-            </a>
-            <a class="inline-block" href="https://sef.discourse.group" target="_blank">SEF Hive</a>
-            <a href="https://github.com/sef-global/sef-site/blob/master/CONTRIBUTING.md" target="_blank">
-                <span class="bg-blue-600 rounded-full p-1 px-2">
+    <nav>
+        <div class="flex flex-1 py-3 md:mt-5 md:mx-20 items-center justify-between">
+    
+            <div class="flex-initial">
+                <p class="font-medium text-lg">DevTeam</p>
+                <p class="text-gray-400 text-sm">Sustainable Education Foundation</p>
+            </div>
+            <div class=" sm:hidden">
+                <!-- Mobile menu button-->
+                <button type="button" class=" float-right text-gray-400 navbar-toggler d-block d-sm-none" data-toggle="collapse" data-target="#navbarMenu" aria-controls="mobile-menu" aria-expanded="false">
+                  <span class="sr-only">Open main menu</span>
+                  <a href="javascript:void(0);" class="icon flex" onclick="squashMobileView()" >
+                    <i class="fas fa-align-justify p-3"></i>
+                  </a>
+                </button>
+              </div>
+            <div class="flex text-xs font-light space-x-5 hidden md:flex">
+                <a class="inline-block px-1 py-2" href="https://github.com/sef-global" target="_blank">Github</a>
+                <a class="inline-block px-1 py-2" 
+                   href="https://join.slack.com/t/sefheadquarters/shared_invite/zt-9w6v9rdu-uJGLrMKLTNFzQEVPC4sbOw" 
+                   target="_blank">
+                     Slack
+                </a>
+                <a class="inline-block px-1 py-2" href="https://sef.discourse.group" target="_blank">SEF Hive</a>
+                <a href="https://github.com/sef-global/sef-site/blob/master/CONTRIBUTING.md" class="inline-flex items-center justify-center  px-4 py-2 text-center
+                 no-underline align-middle transition-all duration-300 ease-in-out bg-blue-600 border-solid rounded-full 
+                 cursor-pointer select-none" target="_blank">
                     Best Practices
-                </span>
-            </a>
-        </div>
-    </div>
+                </a>
+            </div>
+        </div> 
+            </div>
+    
```

Comment:
I feel like there's an additional closing div tag here. Can you please take a look?

Reply:
removed the additional div